extensions/wikia/UserLogin/js/mixins/UserSignup.mixin.js

```diff
@@ -0,0 +1,23 @@
+(function () {
+	'use strict';
+
+	var UserSignupMixin = function () {
+		/**
+		 * Handle marketing email opt-in for different locales
+		 */
+		this.initOptIn =  function (wikiaForm) {
+			require(['usersignup.marketingOptIn'], function (optIn) {
+				optIn.init(wikiaForm);
+			});
+		};
+		/**
+		 * Send country code upon signup
+		 */
+		this.setCountryValue = function (wikiaForm) {
+			var country = Wikia.geo.getCountryCode();
```

Comment:
There's actually a list of accepted globals in the [.jshintrc](https://github.com/Wikia/app/blob/dev/.jshintrc#L59), and `wikia` is one of them.